src/Microsoft.Diagnostics.Runtime/src/DataReaders/Minidump/MinidumpReader.cs

```diff
@@ -38,7 +36,17 @@ public MinidumpReader(string crashDump, Stream stream)
 
             DisplayName = crashDump;
 
-            _minidump = new Minidump(crashDump, stream);
+            if (new FileInfo(crashDump).Length <= (Environment.Is64BitProcess ? 0x100_0000_0000 : 0x1000_0000))
+            {
+                _file = MemoryMappedFile.CreateFromFile(stream, null, 0, MemoryMappedFileAccess.Read, HandleInheritability.None, leaveOpen: false);
+                _stream = _file.CreateViewStream(0, 0, MemoryMappedFileAccess.Read);
```

Comment:
Since we aren't saving `stream` in this case, we should call `stream.Dispose()` on the parameter.

Reply:
The MMF takes the stream and will dispose of it (`leaveOpen: false`).

Reply:
Ahhh I see thanks!